runtime/compiler/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -140,7 +140,7 @@ J9::AheadOfTimeCompile::addSerializationRecord(const AOTCacheRecord *record, con
 #endif /* defined(J9VM_OPT_JITSERVER) */
 
 uintptr_t
-J9::AheadOfTimeCompile::offsetInSharedCacheFromPointer(TR_SharedCache *sharedCache, void *ptr)
+J9::AheadOfTimeCompile::offsetInSharedCacheFromWellKnownClasses(TR_SharedCache *sharedCache, void *ptr)
```

Comment:
I would rather keep the old name because the function body is quite generic and therefore the old name better reflects what the function can do.

Reply:
I could keep it, but I wanted to change the name because in future the only thing this function should be doing is looking up the offset of a well-known classes object in the SCC, so I can override it at the server to consult the JITServer AOT cache. I could make `offsetInSharedCacheFromWellKnownClasses` a new function that wraps the existing `offsetInSharedCacheFromPointer` perhaps?

Reply:
Yes, a new function seems like a good idea. Then the name of the second parameter should be changed to something more appropriately (rather than the generic ptr).